ruby/v3_2_0/wl-build.sh

```diff
@@ -34,9 +38,11 @@ else
 fi
 
 logStatus "Building ruby..."
-make ruby
+make install
 
 logStatus "Preparing artifacts... "
-mv ruby ${WASMLABS_OUTPUT}/bin/ruby.wasm || exit 1
+mv /wlr-rubies/bin/ruby ${WASMLABS_OUTPUT}/bin/ruby-slim.wasm || exit 1
```

Comment:
@ereslibre, is it possible to use `./configure --prefix` here as in the new PHP 8.2.0 build? It avoids the need for a manual installation of the binaries and other assets.

Reply:
I would like to keep the prefix outside of `WASMLABS_OUTPUT` in this case, because `make install` here will install a number of files (e.g. include for extensions, ruby standard library, binaries...), and we are interested in having only the slim and full ruby versions as the result of the build, right?

By setting the `--prefix` as we are doing here, we can then later map the standard library to the expected place with `wasi-vfs`.